index.html

```diff
@@ -6,7 +6,9 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Israel CHIZUNGU Portfolio</title>
-    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Poppins:ital,wght@0,300;0,500;0,600;0,700&display=swap" rel="stylesheet">
+    <link
```

Comment:
Hello Israel, I would like to suggest you adding your linked fonts through the css file, so you can have everything that css related into your `styles.css` file.
To do that you can take de `@import` on the bottom of selected fonts
![image](https://user-images.githubusercontent.com/67355479/199259707-c70dc3bb-dfe0-4d83-97a5-76a449148b85.png)